ResearchKit/Common/ORKAnswerFormat_Internal.h

```diff
@@ -179,6 +179,8 @@ ORK_DESIGNATE_CODING_AND_SERIALIZATION_INITIALIZERS(ORKTimeIntervalAnswerFormat)
 
 @interface ORKTextAnswerFormat ()
 
+@property BOOL isEmail;
```

Comment:
Would `isEmailEntry` be a better name? Or `isEmailAddress`?

And this format is prefered(If you global search `getter=`).
```
@property (nonatomic, assign, getter=isEmailAddress) BOOL emailAddress;
```